src/repositories/friendship.repository.ts

```diff
@@ -9,6 +9,19 @@ export class FriendshipRepository extends Repository {
     return body;
   }
 
+  async showMany(userIds: string[] | number[]) {
+        const { body } = await this.client.request.send({
+            url: `/api/v1/friendships/show_many/`,
+            method: 'POST',
+            form: {
+                _csrftoken: this.client.state.cookieCsrfToken,
+                user_ids: userIds.join(),
```

Comment:
Are you sure it's working case?
```js
[1,2,345].join() // 12345
```
Looks broken

Reply:
Yes 100% works in production for me. I made the edit in javacript on my side so it didnt require the typescript change. This is my working code that I use now:

`async showMany(userIds) {
        const { body } = await this.client.request.send({
            url: `/api/v1/friendships/show_many/`,
            method: 'POST',
            form: {
                _csrftoken: this.client.state.cookieCsrfToken,
                user_ids: userIds.join(),
                _uuid: this.client.state.uuid
            },
        });
        return body.friendship_statuses;
    }`

Heres an example of the input:
`[7445255807, 10000608966, 13252455216, 10801754590]`

and when adding join, makes it (adds a comma delimeter): 
`7445255807,10000608966,13252455216,10801754590`

Reply:
This is where I figured it out from: https://instagram-private-api.readthedocs.io/en/latest/_modules/instagram_private_api/endpoints/friendships.html

Reply:
then it should be `userIds.join(',')`

Reply:
Well since "," is the default delimiter in the join function I find it unnecessary.

Link: [](https://www.geeksforgeeks.org/javascript-array-join-method/)
Says: 
> separator: It is Optional i.e, it can be either used as parameter or not.Its default value is comma(, ).

Reply:
i.e. [1,2,345].join() // 1,2,345

Reply:
@dilame Would you like me to add the comma buddy?

Reply:
Sorry, man, i feature guess I forgot that, or even never knew. You are right